MobiFlight-Installer/MobiFlight-Installer/model/MobiFlightUpdaterModel.cs

```diff
@@ -150,15 +157,18 @@ public static void GoExtractToDirectory(string zipPath, string destinationDirect
 
                     if (file.Name == "MobiFlight-Installer.exe")
                     {
-                        if (InstallerIsNewer(file)) // NewInstaller have greater version than current
+                        if ((!File.Exists(InstallPath + "//" + file.Name)) || InstallerIsNewer(file))  // NewInstaller have greater version than current or installer not exist
```

Comment:
`Path.Join()` again, and elsewhere in this file. I realize some of this is older code but since you are touching it might as well clean it up to the more robust way of joining path parts together.